### PR TITLE
CLDR-17830 BRS Alpha release page (basic Alpha version)

### PR DESCRIPTION
**Pull request by macchiati**
CLDR-17830

This is a base version of the new release page. The TBD sections we will be filling out over time.

Note that it is in Markdown, not Sites. So to review, go to "Files Changed", then click on the 📄 icon (see below) to see basically what it will look like.

Screenshot 2024-08-20 at 16 30 48

 (Like the spec, it will go through some processing to get better styling, links on headers, etc.)

- [ ] This PR completes the ticket.

ALLOW_MANY_COMMITS=true

### CODE REVIEW

**Review by srl295 — APPROVED** [review]

File: docs/site/downloads/cldr-46.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
+# CLDR 46 Release Note
+
+| No. |    Date    | Rel. Note |  Data  |  Charts  | Spec |   Delta  | GitHub Tag | Delta DTD | CLDR JSON |
+|:---:|:----------:|:---------:|:------:|:--------:|:------------:|:---:|:----------:|:---------:|:---------:|
+|  46 | 2024-010-~~XX~~ |    ~~[v46]()~~   | ~~[CLDR46](http://unicode.org/Public/cldr/46/)~~ | [Charts46](http://unicode.org/cldr/charts/dev) |    [LDML46](http://www.unicode.org/reports/tr35/proposed.html)    | [Δ46](https://unicode-org.atlassian.net/issues/?jql=project+%3D+CLDR+AND+status+%3D+Done+AND+resolution+%3D+Fixed+AND+fixVersion+%3D+%2246%22+ORDER+BY+priority+DESC) | ~~[release-46]()~~ |   [ΔDtd46](https://www.unicode.org/cldr/charts/dev/supplemental/dtd_deltas.html)  |   ~~[46.0.0](https://github.com/unicode-org/cldr-json/releases/tag/46.0.0)~~  |
```
Comment (markusicu) [inline docs/site/downloads/cldr-46.md:5]:
```suggestion
|  46 | 2024-10-~~XX~~ |    ~~[v46]()~~   | ~~[CLDR46](http://unicode.org/Public/cldr/46/)~~ | [Charts46](http://unicode.org/cldr/charts/dev) |    [LDML46](http://www.unicode.org/reports/tr35/proposed.html)    | [Δ46](https://unicode-org.atlassian.net/issues/?jql=project+%3D+CLDR+AND+status+%3D+Done+AND+resolution+%3D+Fixed+AND+fixVersion+%3D+%2246%22+ORDER+BY+priority+DESC) | ~~[release-46]()~~ |   [ΔDtd46](https://www.unicode.org/cldr/charts/dev/supplemental/dtd_deltas.html)  |   ~~[46.0.0](https://github.com/unicode-org/cldr-json/releases/tag/46.0.0)~~  |
```

**Review by markusicu — APPROVED** [review]
I will have sometime to add for collation.

**Comment by macchiati** [comment]
Thanks, Markus. I'll go ahead and merge this one and we can pick up that item later.

**Comment by macchiati** [comment]
BTW, now that it is in place, people can use web github to make proposed changes (way better than Sites!)

**Pull request merged**